### PR TITLE
chore: Rename HydroflowPlus to Hydro

### PR DESCRIPTION
**Pull request by rohitkulshreshtha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Hydro provides a high-level language for the majority of developers called [Hydr
 
 ## Development Setup
 
-See the [quickstart section of the Hydroflow+ book](https://hydro.run/docs/hydroflow_plus/quickstart/) for instructions on installing Rust and getting started with the Hydroflow+ template.
+See the [quickstart section of the Hydroflow+ book](https://hydro.run/docs/hydro/quickstart/) for instructions on installing Rust and getting started with the Hydroflow+ template.
```
Comment (jhellerstein) [inline README.md:27]: Needs name change edits. I have a separate PR #1621 with suggested wording, but you'll still need to change the names of directories/files in the docs and links to those in the version in my PR.

File: docs/docs/hydroflow/quickstart/index.md [hunk 2]
```diff
@@ -3,6 +3,6 @@ This section will get you up and running with Rust and Hydroflow, and work you t
 
 :::tip
 
-Hydroflow is the low-level IR that forms the foundation of the Hydro stack. If you're looking for a higher-level API, check out [Hydroflow+](../../hydroflow_plus/index.mdx).
+Hydroflow is the low-level IR that forms the foundation of the Hydro stack. If you're looking for a higher-level API, check out [Hydro](../../hydro/index.mdx).
```
Comment (jhellerstein) [inline docs/docs/hydroflow/quickstart/index.md:6]: Hydroflow => DFIR

**Review by jhellerstein — APPROVED** [review]
Left you comments on merging with #1621 edits to README.md. Otherwise LGTM.

File: docs/docs/hydro/stageleft.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: Stageleft
+sidebar_position: 4
+---
+
+import StageleftDocs from '../../../stageleft/README.md'
+
+Under the hood, Hydroflow uses a library called Stageleft to power the `q!` macro and code generation logic. The following docs, from the Stageleft README, outline the core architecture of Stageleft.
```
Comment (MingweiSamuel) [inline docs/docs/hydro/stageleft.mdx:8]: *Hydro

File: docs/docs/hydroflow/quickstart/index.md (hunk 2 above)
Comment (MingweiSamuel) [reply docs/docs/hydroflow/quickstart/index.md:6]: #1620

**Review by MingweiSamuel — APPROVED** [review]

**Comment by MingweiSamuel** [comment]
Presumably something will break, but we will find out when we need to

File: README.md (hunk 1 above)
Comment (rohitkulshreshtha) [reply README.md:27]: Yep. I will merge #1621 last. 

Thanks for putting that CR together!

**Pull request merged**